foundation/erpnext_foundation/doctype/foundation_fellowship/templates/foundation_fellowship_row.html

```diff
@@ -1,11 +1,16 @@
-<div style='margin-bottom: 30px; max-width: 600px;'
-    class='with-border clickable'>
-    <a href="/{{ doc.route }}">
-        <h3 style='margin-top: 0px;'>{{ doc.title }}</h3>
-        <p>
+<div class="media max-width-medium clickable" style="cursor: pointer;" onclick="location.href='{{ doc.route }}'">
```

Comment:
Check if the is published field checked or not. if it is not then don't show in listing